Fluid/Fluid.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;netstandard2.1;netcoreapp3.1;net5.0;net6.0;net7.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard2.1;net6.0;net7.0;net8.0</TargetFrameworks>
```

Comment:
I told everyone we shouldn't do that ;) Note it was already doing it before though :/

Would love to see if the benchmarks change when a specific target for net8.0 is used. Meaning run benchmarks on net8.0 but with the net8.0 specific tfm in the library, and not.

Reply:
I told you @lahma :)

Reply:
I told you @lahma :)